### ISSUE TITLE
Hyperdevotion Noire: Goddess Black Heart (415480)

### ISSUE THREAD

**Issue by DerUnderpanzer** [ValveSoftware/Proton#762]
it shows this:

 Error

..\..\..\Source\Kernel\Platform\Windows\K_CGraphics.cpp
4108
Kernel::CGraphics::_CreateTexture
unknown
IDirect3DDevice9::CreateTexture() error.

A gist to my System information: 
https://gist.github.com/DerUnderpanzer/2d7067c2689037ad27365cd1019d9588

**Comment by kisak-valve** [comment]
Hello @DerUnderpanzer, please copy your system information from steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report. Additionally, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options and drag and drop the generated $HOME/steam-$APPID.log into the comment box.

**Comment by DerUnderpanzer** [comment]
Sorry my ignorance but where exactly is it located? thank you

**Comment by enimog** [comment]
I have a different error on start for this game ; 
..\..\..\Source\Kernel\Platform\Windows\Application\WinMain.cpp
432
WinMain
SetThreadAffinityMask() error.

Steam informations : [link](https://gist.github.com/enimog/41c45d66ee327a085413468b42265b9c)


[steam-415480.log](https://github.com/ValveSoftware/Proton/files/2386082/steam-415480.log)

**Comment by zzhiyi** [comment]
003a:err:quartz:GetClassMediaFile Media class not found
003a:err:quartz:GetClassMediaFile Media class not found
003a:err:quartz:GetClassMediaFile Media class not found
003a:err:winediag:MIDIMAP_drvOpen No software synthesizer midi port found, Midi sound output probably won't work.
003a:fixme:quartz:Parser_QueryInterface No interface for {37d84f60-42cb-11ce-8135-00aa004bb851}!
003a:fixme:quartz:MPEGSplitter_QueryInterface No interface for {37d84f60-42cb-11ce-8135-00aa004bb851}!
003a:fixme:quartz:MPEGSplitter_query_accept {00000000-0000-0000-0000-000000000000}
003a:fixme:quartz:Parser_QueryInterface No interface for {37d84f60-42cb-11ce-8135-00aa004bb851}!

**Comment by lieff** [comment]
Hmm, {37d84f60-42cb-11ce-8135-00aa004bb851} == IID_IPersistPropertyBag
https://msdn.microsoft.com/ru-ru/office/aa768186(v=vs.90)
What the hell properties they need to load to initialize MPEGSplitter !?

**Comment by zzhiyi** [comment]
To show an intro movie for example.

**Comment by lieff** [comment]
I mean MPEG Splitter it's not like thing that needs configuration. Maybe game do something like load filter graph from file and tries to load properties for all filters despite it's needed or not.

**Comment by HonkingGoose** [comment]
Hi @zzhiyi the Proton log found [here](https://github.com/ValveSoftware/Proton/issues/762#issuecomment-421634128) does not contain the things you mention in: https://github.com/ValveSoftware/Proton/issues/762#issuecomment-478255338

I think I'm missing something here, did your comment come from your own Proton log for the game? If so, can you upload that log, so that I can check and verify your findings?

**Comment by zzhiyi** [comment]
> Hi @zzhiyi the Proton log found [here](https://github.com/ValveSoftware/Proton/issues/762#issuecomment-421634128) does not contain the things you mention in: [#762 (comment)](https://github.com/ValveSoftware/Proton/issues/762#issuecomment-478255338)
> 
> I think I'm missing something here, did your comment come from your own Proton log for the game? If so, can you upload that log, so that I can check and verify your findings?

Yes. It's from my own log. Unfortunately I've already deleted the game. I tested on a self compiled 4.2-1 build. You're welcome to verify it. This is likely related to the missing WMV support in proton.  Checking the game folder for .wmv files and convert them to using different codecs using ffmpeg may help.

**Comment by HonkingGoose** [comment]
@zzhiyi I don't have the game myself, so I can't verify this.

I'm inclined to trust your comment about your log, seeing as you're from Codeweavers. So I can assume you know a lot better than me how to interpret a Proton log... :smile: 

If somebody else has a new Proton log to offer, that would still be helpful to verify though...

I'll add this game to the list at #1464.

**Comment by kah0922** [comment]
[System information](https://gist.github.com/kah0922/12ccf782c927ee2ad14a573f516b94d1)
[steam-415480.log with d9vk](https://github.com/ValveSoftware/Proton/files/3679221/steam-415480.log)
[steam-415480.log](https://github.com/ValveSoftware/Proton/files/3679225/steam-415480.log)

As others have reported, it shows a loading screen and then crashes. It's done this for as long as I've used WINE.

**Comment by kah0922** [comment]
```
6934.920:002f:0041:err:quartz:GetClassMediaFile Media class not found
6934.927:002f:0041:err:quartz:GetClassMediaFile Media class not found
6934.934:002f:0041:err:quartz:GetClassMediaFile Media class not found
```
```
6934.960:002f:0041:fixme:quartz:MPEGSplitter_query_accept {00000000-0000-0000-0000-000000000000}
6934.960:002f:0041:fixme:quartz:MPEGSplitter_query_accept {00000000-0000-0000-0000-000000000000}
```

Of note is that compared to the earlier log, the fixme for MPEGSplitter is different.

**Comment by ipr** [comment]
There is a codec-related issue:

winegstreamer error: decodebin0: Your GStreamer installation is missing a plug-in.
winegstreamer error: decodebin0: ../src-gst_base/gst/playback/gstdecodebin2.c(4719): gst_decode_bin_expose (): /GstBin:bin0/GstDecodeBin:decodebin0:
no suitable plugins found:

Referencing the related tracking issue: #1464

[steam-415480.log](https://github.com/ValveSoftware/Proton/files/8136601/steam-415480.log)

**Comment by kisak-valve** [comment]
Hyperdevotion Noire: Goddess Black Heart (415480)

Issue transferred from https://github.com/ValveSoftware/Proton/issues/5762.
@ComicBookGrunty posted on 2022-04-10T01:14:45:

**Compatibility Report**

Name of the game with compatibility issues: Hyperdevotion Noire: Goddess Black Heart
Steam AppID of the game: 415480

**System Information**

Steam Deck

**I confirm:**

that I haven't found an existing compatibility report for this game.
that I have checked whether there are updates for my system available.

**Symptoms**

The game won't create a save file.

**Reproduction**

Can reboot and play again, choose to save in the menu, no save file is created.  First time reporting anything, hope I did everything right.

**Comment by Renari** [comment]
Save file issue fixed upstream: https://gitlab.winehq.org/wine/wine/-/merge_requests/10946

**Comment by alasky17** [comment]
@Renari Thank you so much for contributing to Wine!  And thanks for the heads up here - I went ahead and cherry-picked the fix into the experimental-11 branch, and it should show up in the bleeding-edge beta branch of Proton - Experimental shortly :)  And then it will make its way into a future Proton - Experimental release.
